python/lsst/pipe/tasks/multiBand.py

```diff
@@ -353,7 +353,11 @@ class DeblendCoaddSourcesConfig(Config):
                                           doc="Deblend sources separately in each band")
     multiBandDeblend = ConfigurableField(target=ScarletDeblendTask,
                                          doc="Deblend sources simultaneously across bands")
-    simultaneous = Field(dtype=bool, default=False, doc="Simultaneously deblend all bands?")
+    simultaneous = Field(dtype=bool,
+                         default=True,
+                         doc="Simultaneously deblend all bands? "
+                             "True uses the 'ScarletDeblendTask' while False performs deblending in "
```

Comment:
I think this docstring should just refer to the ConfigurableField names, not what they are default-configured to. You also need `()` around this docstring, because I'm pretty sure `doc=` is only being assigned to the first line here.

Reply:
I don't think you need parens around string concatenation. Looks fine to me.

Reply:
```
milo:~ $ python xxx.py 
abcdef
milo:~ $ cat xxx.py
def p(x):
    print(x)

p("abc"
  "def")
```

Reply:
Thanks, I never quite figured out when it was and wasn't necessary.